content/play/configuration/_index.md

```diff
@@ -4,4 +4,4 @@ weight: 900
 ---
 
 For most of the use cases, default configuration of Lc0 is good for use.
-If you have NVidia RTX GPU, you may want to change the default network to [one of the larger ones]({{< ref "../network" >}}).
\ No newline at end of file
+If you have NVidia RTX GPU, you may want to change the default network to [one of the larger ones]({{< ref "../networks" >}}).
```

Comment:
Yeah indeed it was broken, thanks for fixing! I fixed it locally but forgot to push.